### ISSUE TITLE
Spontaneous bacterial peritonitis

### ISSUE THREAD

**Issue by johannaest** [obophenotype/human-phenotype-ontology#6177]
**Preferred term label:**
Spontaneous bacterial peritonitis

**Synonyms**
SBP

**Definition (free text, please give PubMed ID)**
The infection of ascitic fluid in the absence of any intra‐abdominal, surgically treatable source of infection.
PMID: 17551068 

**Parent term (use hpo.jax.org/app)**
Peritonitis

**Comment by pnrobinson** [comment]
This is best characcterized as a disease. @nicolevasilevsky I think this probably should be a new Mondo term. @johannaest the use of the word "spontaneous" is superfluous (as suggested in the article itself). Nicole, I think that Mondo needs subterms for bacterial and fungal peritonitis, and Spontaneous bacterial peritonitis could be a related synonym for bacterial peritonitis.
